pkgs/applications/graphics/qosmic/default.nix

```diff
@@ -1,4 +1,4 @@
-{ mkDerivation
+{ stdenv
```

Comment:
not sure if there's any conventions on whether it's better to pull in all of `stdenv` or just the things needed for a particular derivation - this is OK by me.

Reply:
At least in another my PRs reviewers recommended me to use `stdenv.mkDerivation` for Qt apps.

Reply:
Oh man. Every single day a new recommendation.

Usually QT apps are packaged by using `libsForQt.callPackage`, and because of it they use `mkDerivation` instead of `stdenv.mkDerivation`.

Reply:
Just for reference https://github.com/NixOS/nixpkgs/pull/146531#issuecomment-973301105